README.md

```diff
@@ -238,4 +238,11 @@ export PACKAGE={CHART_NAME} # Only need to run once
 make clean
 ```
 
-Ensure that you've already saved your changes with `PACKAGE={CHART_NAME} make patch` and cleaned up your working directory with `PACKAGE={CHART_NAME} make clean`. Then, commit all the remaining changes to `packages/{CHART_NAME}` and submit your PR to the branch `main-source`.
\ No newline at end of file
+Ensure that you've already saved your changes with `PACKAGE={CHART_NAME} make patch` and cleaned up your working directory with `PACKAGE={CHART_NAME} make clean`. Then, commit all the remaining changes to `packages/{CHART_NAME}`.
+
+Once you've committed all your changes in your package directory, run `make charts` and add everything that gets updated to a second commit (Usually `assets`, `charts` and in some cases `index.yaml` as well) so that your Pull Request contents are as following:
```

Comment:
```suggestion
Once you've committed all your changes in your package directory, run `make charts` and add everything that gets updated to a second commit (usually `assets`, `charts` and in some cases `index.yaml` as well) so that your Pull Request's contents are as following:
```